Van_Marukyan/python/perfect_number/perfect.py

```diff
@@ -0,0 +1,10 @@
+def is_perfect_num(n):
+    s = 0
+    for i in range(1, n):
+        if n % i == 0:
+            s = s + i
+    if s == n:
+        print(n, "-is perfect number")
+    else :
+        print(n, "-isn't perfect number")
+is_perfect_num(8128)
```

Comment:
- No any user input
- No any validation for invalid values
![image](https://user-images.githubusercontent.com/80816144/119727406-499dbe80-be83-11eb-86d3-39ed7096bc13.png)